tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
 envlist =
-    py{36,37,38}-test{,-alldeps,-devdeps}{,-cov}
-    py{36,37,38}-test-numpy{116,117,118}
+    py{36,37,38}-test{,-all}{,-dev,-latest,-oldest}{,-cov}
+    py{36,37,38}-test-numpy{116,117,118,119}
     py{36,37,38}-test-scipy{12,13,14,15}
```

Comment:
```suggestion
```

Reply:
These specific pinned versions might be helpful for testing where exactly something breaks, but I also see why less might be more. Either way ok with me, but they do serve a purpose.

Reply:
OK sure lets keep them for now then